packages/firestore/src/local/simple_db.ts

```diff
@@ -692,7 +715,35 @@ function wrapRequest<R>(request: IDBRequest): PersistencePromise<R> {
     };
 
     request.onerror = (event: Event) => {
-      reject((event.target as IDBRequest).error!);
+      const error = (event.target as IDBRequest).error!;
+      checkForAndReportiOSError(error);
+      reject(error);
     };
   });
 }
+
+// Guard so we only report the error once.
+let reportedIOSError = false;
+function checkForAndReportiOSError(error: DOMException): void {
+  if (reportedIOSError) {
+    return;
+  }
+  const iOSVersion = SimpleDb.getIOSVersion(getUA());
+  if (iOSVersion >= 12.2 && iOSVersion < 13) {
```

Comment:
1) Now I understand why we use a version number and not a version number string.
2) Do we need to guard this check on the iOS version? We might just want to log the console message whenever we encounter the matching error string.

Reply:
Given the iOS error message is so generic, I am a little worried about making this check too open-ended and possibly having it hide future bugs in iOS Safari's IndexedDB implementation (I also named this "IOS_INDEXEDDB_BUG1" to allow for future bugs in the future 😛)...  So I prefer keeping the version scoped for now.

Reply:
Sounds good. Glad you didn't use IOS_INDEXEDDB_BUG_001 :)